README.md

```diff
@@ -59,6 +64,6 @@ You can index into the results of these functions like any other Julia composite
 
 Each function is vectorized using `Base.@vectorize_1arg`, so passing an array of user-agent strings will return an array. Finally, DataFrame methods have been defined for arrays of each UAParser type, so calling `DataFrame(DeviceResult)` will transform the array to a DataFrame.
 
-##Licensing
+## Licensing
 
-The data contained in regexes.yaml is Copyright 2009 Google Inc. and available under the Apache License, Version 2.0. The licensing of the UAParser Julia module is under the [default MIT Expat license](https://github.com/JuliaWeb/UAParser.jl/blob/master/LICENSE.md).
+The licensing of the UAParser Julia module is under the [default MIT Expat license](https://github.com/JuliaWeb/UAParser.jl/blob/master/LICENSE.md).
```

Comment:
Please retain the Google and Apache notices. While they may be redundant, IMO it's better to overcredit than to assert that these files can be relicensed.